packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -293,6 +300,26 @@ function SingleForm({ form, appUrl, Page, enrichedWithUserProfileForm }: SingleF
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [form]);
 
+  useEffect(() => {
+    if (form.teamId && form.settings?.sendUpdatesTo?.length && teamMembers?.length) {
+      let sendToAll = true;
+      teamMembers.forEach((member) => {
+        if (!form.settings?.sendUpdatesTo?.includes(member.id)) {
+          sendToAll = false;
+          return;
+        }
+      });
+      setAssignAllTeamMembers(sendToAll);
+      setSelectedMembers(
+        form.settings.sendUpdatesTo.map((userId) => ({
+          isFixed: true,
+          userId: userId,
+          priority: 1,
+        }))
+      );
+    }
+  }, [form.teamId, form.settings?.sendUpdatesTo?.length, teamMembers?.length]);
```

Comment:
Could we instead add to the form a `sendToAll` and a `sendUpdatesTo` field?

Reply:
Done